umount_aufs() hangs 20mn sporadicly.
It may keep /dev/shm/*root mounted

--- a/cook
+++ b/cook
@@ -1011,8 +1011,9 @@
 
 umount_aufs() {
 	tac ${1}rw/aufs-umount.sh | sh
+	umount -l ${1}root 2> /dev/null
+	umount -l ${1}root 2> /dev/null
 	rm -rf ${1}rw
-	umount ${1}root
 	rmdir ${1}r*
 }
 
